### DISCUSSION TITLE
簡介 Angular Signals | 全端開發人員天梯

### DISCUSSION THREAD

**Discussion by giscus** [wellwind/wellwind.github.io#89] [Announcements]
# 簡介 Angular Signals | 全端開發人員天梯

Angular 16 推出了 Angular Signals 功能，雖然還在 developer preview，但個人覺得是個非常值得開始學習，並在未來正式版後開始使用的功能，這篇文章就來介紹一下 Angular Signals 的用法。

https://fullstackladder.dev/blog/2023/05/07/introduction-angular-signals/

**Comment by moose0629** [comment]
文章內有錯字：
> 這樣的「錯」法看起來很聰明

**Reply by wellwind** [reply]
感謝提醒，已修正囉
